Welcome to the resources webpage for the 2016 Integrative Hippocampus Module! We are really excited to spend the next two weeks with you! Please use this webpage as a community resource where you will find links to lessons, protocols, useful tools, and information about module and the instructors. A description of the overall module objectives and goals, [visit this page](ModuleOverview.md). 

## Meet the Instructors
During this module, you will learn a lot from [JuanMarcos](http://www.alarconlab.com), [Eva Fischer](http://evakfischer.weebly.com), [Rayna Harris](http://raynamharris.github.io),
[Rachel Wright](http://rmwright.weebly.com), [Hans Hofmann](http://cichlid.biosci.utexas.edu), and [Andre Fenton](http://www.cns.nyu.edu/corefaculty/Fenton.php). You can also read our short biographies [here](Instructors.md).

## Mouse qPCR Project 
The files for this project are stored on the server. They can also be accessed [here](https://github.com/raynamharris/qPCR-mouse). 


## Week 1 Schedule

Day | Activity|Where
:---|:---|:---
Mon | Morning Lectures | Loeb G70N
 | Module Overview & Introductions | TBA
 | [Hippocampus Image Analysis](https://github.com/raynamharris/qPCR-mouse/tree/master/slicephotos) and [Getting a TACC account](https://github.com/raynamharris/IntegrativeSTGmodule2016) | Lillie 103
 | [Introduction R for Data Organization, Analysis, & Graphing](https://github.com/raynamharris/R_Intro_for_Bioinformatics) | Lillie 103
Tues | Morning Lectures | Loeb G70N
 | Discuss Hippocampus Candidate Genes | TBA
 | [Pipette exercise: RNA dilution series](protocols/RNA_pipette_exercise.md) and [qPCR exercise: 18S standard curve](protocols/qPCRexercise.md) |  Loeb 157
 | Principles of qPCR, analysis of 18S standard curves, Primer Design for qPCR | Lillie 103
Wed | No Morning Lectures | 
 | [Taqman qPCR - Prkci, Prkcz, Frm1, Arc Standard Curves](protocols/aqManqPCRProtocol.pdf) and [Analysis of qPCR data in R with MCMC.qpcr](mcmc.qpcr.tutorial.pdf)| Loeb 157
 | [RNA isolation](protocols/RNAisolationProtocol.pdf) and [Reverse Transcription](protocols/ReverseTranscriptionProctocol.pdf) of whole slices | Loeb 157 
 | Wednesday evening lecture, discussion of special projects| Rowe, breakroom
Thurs | Morning Lectures | Loeb G70N
 | [RNA isolation](protocols/RNAisolationProtocol.pdf) from punches | RMW RMH 
 | [Reverse Transcription](protocols/ReverseTranscriptionProctocol.pdf) of punches. [Taqman qPCR - Prkci, Prkcz, Frm1, Arc on tissues and punches](protocols/TaqManqPCRProtocol.pdf) | Loeb 157  
 | Data analysis and interpretation in R, order primers  | Loeb 157
Fri | Morning Lectures | Loeb G70N
 | Installing software | Breakfroom
 |[Sybr Green qPCR on new genes: standard curves, slices, and punches](protocols/SybrGreenqPCRProtocol.pdf) | Loeb 157
 | Data analyis & Friday evening lecture | Lillie, breakfoom
Sat | Morning Lectures |  Loeb G70N
 | Wrapup | Loeb 157
 | The Usual Mandatory Recreation + Fun time on the beach! | Outside!


## Week 2 Schedule
Day | Activity|Where
:---|:---|:---
Mon | Morning Lectures | Loeb G70N
 | Intro to Transcriptomic Datasets | TBD
 | Overview of Next Generation Sequencing, [Intro to Unix](https://github.com/raynamharris/Shell_Intro_for_Transcriptomics), [Intro to TACC](https://github.com/raynamharris/IntegrativeSTGmodule2016) | Lillie 100a
 | Transcriptomics Part 1: de novo transcriptome constuction | Lillie 100a
Tues | Morning Lectures | Loeb G70N
 | Experimental design considerations | TBD
 | Transcriptomics Part 2: alignment, abundance estimation, DE analysis | Lillie 103
 | Transcriptomics Part 3: Multivariate and Covariance analyses | Lillie 103
Wed- Fri | Work on Group Projects | Loeb 157
Sat | Group Presentations |  Loeb G70N
 | The Usual Mandatory Recreation + Fun time on the beach! | Outside!
 | End of Course Party!| TBD


